templates/guides/get-started.md

```diff
@@ -0,0 +1,216 @@
+---
+page_title: "Get started with Astro Terraform Provider"
+---
+
+# Get started with Astro Terraform Provider
+In this guide, we will automate the onboarding of a new team by creating and managing an Astro Workspace and Deployment. By the end of this tutorial, you will have a fully automated setup that is reproducible and easily scalable to more teams.
+
```

Comment:
I think we might need a pre-requisites section. 

so that would include:
- installing terraform
- Astro organization, workspace, deployment, users etc?

Reply:
I agree will add
- [Terraform](https://developer.hashicorp.com/terraform/install)
- An Astro Organization
Workspace and deployment are created with TF in this tutorial